zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/socket/Message.java

```diff
@@ -128,6 +128,7 @@
 
     LIST_NOTEBOOK_JOBS,     // [c-s] get notebook job management infomations
     LIST_UPDATE_NOTEBOOK_JOBS, // [c-s] get job management informations for until unixtime
```

Comment:
I think `LIST_UPDATE_NOTEBOOK_JOBS` is server -> client message. And it sends updated job informations after the given unixtime.
So, comment should be something like `[s-c]  update list of jobs since given unix time.`
And it's got `lastResponseUnixTime` and `jobs` in it's param. Could you add params in the comment?

And i think `UPDATE_NOTEBOOK_JOBS` for key name is less confusing.